test cases/rust/12 bindgen/meson.build

```diff
@@ -90,6 +90,32 @@ rust_bin2 = executable(
 
 test('generated header', rust_bin2)
 
+# Test generating a static inline wrapper
```

Comment:
I would suggest making this use an option + a matrix in test.json, so that the tests can specifically check for new-bindgen-stuff.